components/org.wso2.carbon.identity.oauth.ciba/src/main/java/org/wso2/carbon/identity/oauth/ciba/dao/CibaAuthMgtDAOImpl.java

```diff
@@ -0,0 +1,547 @@
+/*
```

Comment:
On a general note, we do not need to include HTTP status code in the exceptions thrown at this layer.

Reply:
It would be better to use extended exception types along with defined error codes

Reply:
Ack.

Reply:
Refactored.